config/locales/es.yml

```
@@ -1140,9 +1140,13 @@
       title: "Tiene una nueva oportunidad de mejora"
       oportunity_link: "Ver oportunidad de mejora"
     notify_new_finding_answer:
-      title: "Nuevo comentario en el informe %{review}"
+      title: "Nuevo comentario en el informe %{review} [#%{finding_id}]"
       finding_data: "Se ha agregado un comentario al hallazgo **%{finding_code}** del informe **%{review}**"
       finding_link: "Ver hallazgo"
+    notify_not_added_answer:
+      title: "No se ha podido agregar el comentario que usted esta enviando"
+      warning: "No se ha podido agregar el comentario que usted esta enviando ya que no estamos recibiendo el mismo asunto en la respuesta, 
+                por favor veriquelo."
```

```suggestion
      warning: "No se tomó acción con el correo recibido."
```
Le pondría algo así escueto porque vamos a dejar esta respuesta para cualquier caso, ahora es solo la respuesta de observaciones... pero seguro después nos piden más similares.

---

dale genial